iOS/LocalPackages/DataBrokerProtection-iOS/Sources/DataBrokerProtection-iOS/Bundle/DataBrokerProtectionBundleExtension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  DataBrokerProtectionBundleExtension.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public extension UserDefaults {
+    static let dbp = UserDefaults(suiteName: "dbp")!
```

Comment:
This is possibly worth a discussion. I vaguely remember we were trying to avoid the "com.duck...etc" format?

Reply:
Yeah I recall this too, can't remember where we got to though - definitely need some kind of expectation around this. Tbh though this works for now, as long as it's easy to tell from the name what it's used for then I'm happy.